src/components/GlobalNavigation/index.tsx

```diff
@@ -1,11 +1,17 @@
-import { NavBar, Left, Middle, Right } from './styles';
+import logo from '~/assets/logo.png';
+import { NavBar, Side, Center, Logo, LogoLink } from './styles';
 
 export default function GlobalNavigation() {
   return (
     <NavBar>
-      <Left />
-      <Middle />
-      <Right />
+      <Side>
+        <LogoLink to="/">
+          <Logo alt="Logo" src={logo} />
+          Bookdart
+        </LogoLink>
+      </Side>
+      <Center />
+      <Side />
```

Comment:
I think GlobalNavigation need to contain logo.
Why did you separate Side-Center-Side?
Does it need to match main component?

Reply:
I don't understand what "matching with main component" means, but it's widely used design.